ideas/127-SOB-testing-process.md

```diff
@@ -0,0 +1,44 @@
+
+## Preamble
+
+    Idea: 127
+    Title: Improve OpenBounty Testing and make it easy to contribute
+    Status: In Progress
+    Created: 2018-04-02
+
+## Summary
+- make QA process in SOB team easy to understand and support; 
+- in order  to reduce time spent on regression testing automate main use cases;
+
+## Swarm Participants
+- Lead Contributor:@churik
+- Testing & Evaluation: @annadanchenko 
+- Contributor: @asemiankevich 
+
+## Product Overview
+Full and transparent QA process in Status Open Bounty team will help to involve new people into the testing process more quickly (if needed) and provide developers with more info about what was tested in particular PR or in `develop` branch. Ideally process should be integrated with TestRail, but initially, a smoke checklist should be defined and then automated.
+
+All regression test cases should be automated eventually in order to reduce time for SOB testing and to find issues faster.
+In future would be nice to integrate GitHub pull requests with automation testing thus after PR will be built and deployed,  it will trigger end-to-end tests and publish test results in PR comment.
+Hence this idea is a placeholder for current and future iterations of improving QA process in SOB team.
+
```

Comment:
Could we add success metrics and exit criteria?

Reply:
@oskarth added, but fairly I'm not sure that it can be measured in a particular way. Generally, it should affect testing speed and that's it.